Library/Homebrew/shims/super/cc

```diff
@@ -270,6 +270,7 @@ class Cmd
   end
 
   def cppflags
+    isystem = mac? ? "-isystem" : "-idirafter"
     path_flags("-isystem", isystem_paths) + path_flags("-I", include_paths)
```

Comment:
What are `isystem_paths`? Is there any path other than `HOMEBREW_PREFIX/include`?

Reply:
It currently includes `HOMEBREW_PREFIX/include` and the include directories of all of the `xorg` packages, if `xorg` is installed (and whether it's a dependency or not of the formula being installed).